docs/samples/v1beta1/spark/sparkml.py

```diff
@@ -0,0 +1,17 @@
+from pyspark.ml import Pipeline
+from pyspark.ml.classification import DecisionTreeClassifier
+from pyspark.ml.feature import RFormula
+
+df = spark.read.csv("Iris.csv", header = True, inferSchema = True)
+
+formula = RFormula(formula = "Species ~ .")
+classifier = DecisionTreeClassifier()
+pipeline = Pipeline(stages = [formula, classifier])
+pipelineModel = pipeline.fit(df)
+
+from pyspark2pmml import PMMLBuilder
+
+pmmlBuilder = PMMLBuilder(sc, df, pipelineModel)
+
+pmmlBuilder.buildFile("DecisionTreeIris.pmml")
```

Comment:
can the example directly dump to gcs?

Reply:
> can the example directly dump to gcs?

No don't think the pmml API supports that.